README.md

```diff
@@ -21,8 +21,8 @@ Before you will start with documentation check our demo app: [Ember-Custom-Actio
 
 ### Model actions
 To define custom action like: `posts/1/publish` you can use
-`modelAction(path, options)` method with arguments:
-- `actionId/path` -  if you want to override methods like `urlForCustomAction` use it as `actionId` otherwise, use it as url of the action (in our case it's `publish`)
+`modelAction(actionId/path, options)` method with arguments:
+- `actionId/path` -  if you want to override methods like `urlForCustomAction` (more details in [Adapter customization](#Adapter-customization)) use it as `actionId` otherwise, use it as url of the action (in our case it's `publish`)
```

Comment:
Wouldn't be better to mention here adapter's integration? Eg.

> if you want to integrate it with adapter/serializer use it as `actionId`

Reply:
I've changed the info.  

Or it could be: 
```
modelAction(actionId, options) ...
actionId - use it as relative path of the action (in our case it's `publish`). You can also use it during adapter/serializer integration (more details in [Adapter customization](#Adapter-customization)) to distinguish the actions
...
```